content/sensu-go/5.17/reference/secrets.md

```diff
@@ -0,0 +1,213 @@
+weight: 145
+version: "5.17"
+product: "Sensu Go"
+menu: 
+  sensu-go-5.17:
+    parent: reference
+---
+
+- [Secret specification](#secret-specification)
+  - [Top-level attributes](#top-level-attributes) | [Metadata attributes](#metadata-attributes) | [Spec attributes](#spec-attributes)
+- [Secret configuration](#secret-configuration)
+- [Secret payload examples](#secret-payload-examples)
+
+**COMMERCIAL FEATURE**: Access the Secret datatype in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][1].
+
+Sensu's secrets management eliminates the need to expose secrets in your Sensu configuration.
+Use secrets to obtain secrets from one or more external secrets management providers, refer to external secrets, and comsume secrets via backend [environment variables][5].
+
+Only Sensu backends have access to request secrets from a [secret provider][7].
+Sensu backends cache fetched secrets in memory, with no persistence to a Sensu datastore or file on disk.
+Secrets provided via a "lease" with a "lease duration" are deleted from Sensu's in-memory cache after the configured number of seconds, prompting the Sensu backend to request the secret again.
+
+Secrets are only transmitted over a TLS websocket connection.
+Unencrypted connections must not transmit privileged information.
+For agent-side resources, enable TLS/mTLS.
+
+Secrets are only exposed to Sensu services like environment variables and are automatically redacted from all logs, the API, and the Sensu dashboard.
+ 
+## Secret specification
+
+### Top-level attributes
+
+type         | 
+-------------|------
+description  | Top-level attribute that specifies the resource type. For secrets configuration, the type should always be `Secret`.
+required     | Required for secrets configuration in `wrapped-json` or `yaml` format.
+type         | String
+example      | {{< highlight shell >}}"type": "Secret"{{< /highlight >}}
+
+api_version  | 
+-------------|------
+description  | Top-level attribute that specifies the Sensu API group and version. For secrets configuration in this version of Sensu, the `api_version` should always be `secrets/v1`.
+required     | Required for secrets configuration in `wrapped-json` or `yaml` format.
+type         | String
+example      | {{< highlight shell >}}"api_version": "secrets/v1"{{< /highlight >}}
+
+metadata     |      |
+-------------|------
+description  | Top-level scope that contains the secret's `name` and `namespace`.
+required     | true
+type         | Map of key-value pairs
+example      | {{< highlight shell >}}
+"metadata": {
+  "name": "sensu-ansible-token",
+  "namespace": "default"
+}
+{{< /highlight >}}
+
+spec         | 
+-------------|------
+description  | Top-level map that includes secrets configuration [spec attributes][12].
```

Comment:
This link didn't render correctly:
<img width="876" alt="Screen Shot 2020-01-21 at 3 02 26 PM" src="https://user-images.githubusercontent.com/10634078/72850906-0e6ff780-3c5f-11ea-9cd8-128b781a2889.png">

Reply:
Sorry about that -- I forgot to add the link.